lib/debuglet.js

```diff
@@ -341,6 +341,13 @@ Debuglet.prototype.addBreakpoint_ = function(breakpoint, cb) {
       }
 
       that.logger_.info('Breakpoint hit!: ' + breakpoint.id);
+      if (breakpoint.action === 'LOG') {
+        var message = Debuglet.format(breakpoint.logMessageFormat,
+          breakpoint.evaluatedExpressions.map(function(v) {
+            return v.value;
+          }));
+        console.log(message);
+      }
       that.completeBreakpoint_(breakpoint);
```

Comment:
Should log points be one-shot? Are they supposed to be completed on the server and deleted? I would expect they would stick around for some amount of time.